sentences/pl/_common.yaml

```diff
@@ -76,38 +94,258 @@ lists:
+      - in: "zasłon[a|y]"
+        out: "curtain"
+      - in: "drzwi"
+        out: "door"
+      - in: "(garaż(e)|brama garażowa|(drzwi|bramy) garażowe|(drzwi|brama|bramy) do garażu)"
+        out: "garage"
+      - in: "bram[a|y]"
+        out: "gate"
+      - in: "osłon[a|y]"
+        out: "shade"
+      - in: "żaluzj(i|a|e)"
+        out: "shutter"
+      - in: "ok(ien|no|na)"
+        out: "window"
+      - in: "klap[a|y]"
+        out: "damper"
   lock_states:
     values:
       - in: "zablokowan(e|y|ych)|zamknięt(y|e|ych)"
         out: "locked"
       - in: "odblokowan(e|y|ych)|otwart(e|y|ych)"
         out: "unlocked"
 
+  # binary_sensor
+  bs_battery_states:
+    values:
+      - in: "niski"
+        out: "on"
+      - in: "normalny"
+        out: "off"
+
+  bs_battery_charging_states:
+    values:
+      - in: "ładuje się"
+        out: "on"
+      - in: "nie ładuje się"
+        out: "off"
+
+  bs_carbon_monoxide_states:
+    values:
+      - in: "(wykryty|wyzwolony|włączony)"
+        out: "on"
+      - in: "brak"
+        out: "off"
+
+  bs_cold_states:
+    values:
+      - in: "zimno"
+        out: "on"
+      - in: "normalnie"
+        out: "off"
+
+  bs_connectivity_states:
+    values:
+      - in: "połączony"
+        out: "on"
+      - in: "rozłączony"
+        out: "off"
+
+  bs_door_states:
+    values:
+      - in: "otwarte"
+        out: "on"
+      - in: "zamknięte"
+        out: "off"
+
+  bs_garage_door_states:
+    values:
+      - in: "otwarty"
+        out: "on"
+      - in: "zamknięty"
+        out: "off"
+
+  bs_gas_states:
+    values:
+      - in: "(wykryty|wyzwolony|włączony)"
+        out: "on"
+      - in: "brak"
+        out: "off"
+
+  bs_heat_states:
+    values:
+      - in: "gorący"
+        out: "on"
+      - in: "normalny"
+        out: "off"
+
+  bs_light_states:
+    values:
+      - in: "świecący"
+        out: "on"
+      - in: "brak światła"
+        out: "off"
+
+  bs_lock_states:
+    values:
+      - in: "odblokowany"
+        out: "on"
+      - in: "zablokowany"
+        out: "off"
+
+  bs_moisture_states:
+    values:
+      - in: "mokro"
+        out: "on"
+      - in: "sucho"
+        out: "off"
+
+  bs_motion_states:
+    values:
+      - in: "(wykryty|wyzwolony|włączony)"
+        out: "on"
+      - in: "brak"
+        out: "off"
+
+  bs_occupancy_states:
+    values:
+      - in: "(wykryty|wyzwolony|włączony)"
+        out: "on"
+      - in: "brak"
+        out: "off"
+
+  bs_opening_states:
+    values:
+      - in: "otwarty"
+        out: "on"
+      - in: "zamknięty"
+        out: "off"
+
+  bs_plug_states:
+    values:
+      - in: "podłączony"
+        out: "on"
+      - in: "odłączony"
+        out: "off"
+
+  bs_power_states:
+    values:
+      - in: "(zasilany|podłaczony)"
+        out: "on"
+      - in: "(nie zasilany|wyłączony)"
+        out: "off"
+
+  bs_presence_states:
+    values:
+      - in: "(w domu|obecny)"
+        out: "on"
+      - in: "(poza domem|nie obecny)"
```

Comment:
It should be `nieobecny` - https://polszczyzna.pl/nieobecna-czy-nie-obecna-jak-poprawnie-zapisac-to-slowo/

Reply:
💪🏻 improved